lab_4_retrieval_w_clustering/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 8
```

Comment:
You just forgot to change score to 10. Isn't that right?

Reply:
THAT'S NOT RIGHT. I would do at 10 because of principle if I did not fell ill and under sleep heavily, I would really implement it! Moreover, I do not see the point: EVEN if I open my lab3 PR to higher the mark from 8 to 10 and also make the 4th lab at 10, I will get ONLY 6,42 (including my 100% attendance). I will have 6,6 ONLY if I will also do the 2nd lab at 10 (currently I have 9)... Three improvements seems to be impossible right now, honestly. I would be really excited to get 6,6 which would appear, undoubtedly, to be 7! I would be EXTREMELY calm at the exam just having fun with **the main boss and his dear colleagues...** But I cannot... Otherwise, I will probably go insane by the nearest Friday

Reply:
To be honest, I feel guilty by not doing my best. I will think a little about all that stuff and probably bury myself in grief... Anyways, thank you for believing in my intellectual abilities